packages/vue/examples/data-search/src/App.vue

```diff
@@ -0,0 +1,62 @@
+<template>
```

Comment:
There are linting issues in this file. Please verify if ESLint is working properly for this setup.